bin/cradle

```diff
@@ -88,6 +88,28 @@ def getArgs():
 	normalize_optional.add_argument('-p', type=int, help="The number of cpus. default=(available cpus)/2", required=False)
 	normalize_optional.add_argument('-o', help="Output directory. All normalized bigwig files will be stored here. If the directory doesn't exist, cradle will make the directory. default=CRADLE_normalization.", required=False, default="CRADLE_normalization")
 
+
+	########### covariates
+	covariate_parser = subparsers.add_parser("covariates", help="Calculate covariate values")
+
+	covariate_required = covariate_parser.add_argument_group("Required Args")
+	covariate_required.add_argument('-l', type=int, help="Fragment length.", required=True)
+	covariate_required.add_argument('-r', help="Text file that shows regions of analysis. Each line in the text file should have chromosome, start site, and end site that are tab-spaced. ex) chr1\t100\t3000", required=True)
+	covariate_required.add_argument('-biasType', help="Type of biases you want to correct among 'shear', 'pcr', 'map', 'gquad'. If you want to correct 'shear' and 'pcr' bias, you should type -biasType shear pcr. If you type map, -mapFile and -kmer are required. If you type gquad, -gquadFile is required", nargs='+', required=True)
+	covariate_required.add_argument('-genome', '-faFile', help=".2bit file.", required=True) # faFile is deprecated
+
+	covariate_optional = covariate_parser.add_argument_group("Optional Args")
+	covariate_optional.add_argument('-binSize', type=int, help="The size of bin (bp) for correction. If you put '1', it means you want to correct read counts in single-bp resolution. (default=1)", default=1)
```

Comment:
I think we need to remove this parameter. I believe the current `correctBias_stored` assumes the pre-computed covariates are in single-bp resolution. If users use different values other than 1 and try to use correctBias_stored, it will cause a problem.